themes/_default.yaml

```diff
@@ -530,17 +530,17 @@ sliding-panel:
     opacity: "1"
     position: "absolute"
     top: "0"
-    transition: "transform 195ms cubic-bezier(0.0, 0.0, 0.2, 1)"
+    right-animation: "sidePanelClose 195ms ease-in-out 0s 1 both;"
```

Comment:
if you do not have different values for the other brands, pls do not add this as vars in yaml. :)